grakn-bootup/src/main/java/ai/grakn/bootup/AbstractProcessHandler.java

```diff
@@ -33,44 +33,34 @@
  */
 public abstract class AbstractProcessHandler {
 
-    public final static long WAIT_INTERVAL_S=2;
+    public static final long WAIT_INTERVAL_S=2;
+    public static final String BIN = "/bin/sh";
 
     public OutputCommand executeAndWait(String[] cmdarray, String[] envp, File dir) {
 
         StringBuilder outputS = new StringBuilder();
         int exitValue = 1;
 
         Process p;
-        BufferedReader reader = null;
         try {
             p = Runtime.getRuntime().exec(cmdarray, envp, dir);
             p.waitFor();
             exitValue = p.exitValue();
-            reader =
-                    new BufferedReader(new InputStreamReader(p.getInputStream(), StandardCharsets.UTF_8));
-
-            String line;
-            while ((line = reader.readLine()) != null) {
-                outputS.append(line).append("\n");
+            try(BufferedReader reader = new BufferedReader(new InputStreamReader(p.getInputStream(), StandardCharsets.UTF_8))){
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    outputS.append(line).append("\n");
+                }
             }
-
         } catch (InterruptedException | IOException e) {
             // DO NOTHING
-        } finally {
-            if (reader != null) {
-                try {
-                    reader.close();
-                } catch (IOException e) {
-                    // DO NOTHING
-                }
-            }
         }
         return new OutputCommand(outputS.toString().trim(), exitValue);
     }
 
     public Optional<String> getPidFromFile(Path fileName) {
         String pid=null;
-        if (Files.exists(fileName)) {
+        if (fileName.toFile().exists()) {
```

Comment:
Did Sonarcloud say to do this one? Just curious.

Reply:
Yep. Apparently `File.exists` is VERY slow with JDK 8